packages/wordpress-block-editor-config/src/settings/__experimental-fetch-link-suggestions.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * WordPress dependencies
+ */
+// @ts-ignore TODO
+import apiFetch from '@wordpress/api-fetch';
```

Comment:
🤔 apiFetch doesn't return a type definition yet, let me know if folks have ideas on a good path forward for this.

Reply:
Any thoughts @sarayourfriend @sirreal @dmsnell ? Feel free to tag anyone else who might have ideas.

Reply:
One solution is that you could add a `types.d.ts` that defines very basic types for `apiFetch` which would be better than nothing like:

```ts
declare module '@wordpress/api-fetch' {
	export default function( request: { path: string } ): Promise< any >;
}
```

Then add a `todo` at the top of that file to remove it once `api-fetch` is fully typed.

Alternatively just disable type-checking for this package until `api-fetch` generates its own type definitions and add a note to #18838 to update this package when `api-fetch` is finished (add this to the `api-fetch` entry there).

Reply:
Thanks for taking a look @sarayourfriend. I'll try the partial definition first

Reply:
I've added the partial definition in 9cfa345 👍  if this lands, I'll add a comment to https://github.com/WordPress/gutenberg/issues/18838